### PR TITLE
Add professors of UESTC

### PR DESCRIPTION
**Pull request by Xovee**
**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts. Make sure your GitHub profile contains your full name.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

**Comment by Xovee** [comment]
Dear Prof. Emery Berger, 

I've traversed the CS faculty catalog of UESTC, adding new professors and correcting names (mainly adding 0001 to previous names). 

Thanks for your efforts in maintaining this great service. Have a good day!

Best,

Xovee Xu
PhD candidate in CS
UESTC, Chengdu, China

**Comment by Xovee** [comment]
The homepages I provided are valid official websites but some of them did not pass the homepage checks (validate_commit.py) -- possibly because of the (temporary) firewalls of the university.

**Comment by Xovee** [comment]
@emeryberger Dear Professor, I have made necessary changes for passing all checks.

File: csrankings-z.csv [hunk 1]
```diff
@@ -110,6 +111,7 @@ Zheng Shou 0001,National University of Singapore,https://sites.google.com/view/s
 Zheng Wang 0001,University of Leeds,https://eps.leeds.ac.uk/computing/staff/6452/dr-zheng-wang,qJ7ZKG8AAAAJ
 Zheng Wang 0007,Wuhan University,https://cs.whu.edu.cn/info/1019/2854.htm,-WHTbpUAAAAJ
 Zheng Wang 0025,University of Southern Mississippi,http://www.cs.miami.edu/home/zwang,NOSCHOLARPAGE
+Zheng Wang 0044,UESTC,https://www.scse.uestc.edu.cn/info/1081/11217.htm,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-z.csv]: 404 on this web page.

File: csrankings-z.csv [hunk 2]
```diff
@@ -74,6 +74,7 @@ Zhaojun Bai,Univ. of California - Davis,http://www.cs.ucdavis.edu/~bai,soiDQbQAA
 Zhaolei Zhang,University of Toronto,http://sites.utoronto.ca/zhanglab,WvJIvxQAAAAJ
 Zhaonian Zou,Harbin Institute of Technology,http://homepage.hit.edu.cn/zou,KMYONIEAAAAJ
 Zhaoping Li 0001,University of Tübingen,https://webdav.tuebingen.mpg.de/u/zli/bio.html,V3sgCS0AAAAJ
+zhaoqiang Liu,UESTC,https://sites.google.com/site/liuzhaoqiang2016/,EmGrPbIAAAAJ
```
Comment (emeryberger) [inline csrankings-z.csv]: Lower-case name incorrect

File: csrankings-h.csv [hunk 3]
```diff
@@ -72,7 +72,7 @@ Haidar M. Harmanani,Lebanese American University,http://sas.lau.edu.lb/csm/peopl
 Haifeng Liu 0006,Zhejiang University,http://mypage.zju.edu.cn/hfliu,oW108fUAAAAJ
 Haifeng Xu,University of Chicago,https://cs.uchicago.edu/people/haifeng-xu/,nLgg388AAAAJ
 Haifeng Yu,National University of Singapore,https://www.comp.nus.edu.sg/~yuhf,IDxePZ0AAAAJ
-Haigang Gong,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/2031.htm,NOSCHOLARPAGE
+Hai-gang Gong,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/2031.htm,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings-h.csv:76]: Both entries need to be present.

Comment (emeryberger) [reply csrankings-h.csv:76]: (Haigang Gong is an alias)

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-z.csv (hunk 1 above)
Comment (Xovee) [reply csrankings-z.csv]: I can no longer find "Zheng Wang 0044" in the CS school, therefore I removed this entry.

File: csrankings-z.csv (hunk 2 above)
Comment (Xovee) [reply csrankings-z.csv]: Corrected this.

File: csrankings-h.csv (hunk 3 above)
Comment (Xovee) [reply csrankings-h.csv:76]: Both entries are included.

**Review by Xovee — COMMENTED** [review]
Dear Professor, 

Appreciated for your careful checks. All issues have been addressed. @emeryberger

**Pull request merged**